docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+module.exports = {
+    baseUrl: '/idearium-lib/',
+    favicon: 'img/favicon.ico',
+    onBrokenLinks: 'throw',
+    onBrokenMarkdownLinks: 'warn',
+    organizationName: 'idearium',
+    presets: [
+        [
+            '@docusaurus/preset-classic',
+            {
+                docs: {
+                    editUrl:
+                        'https://github.com/idearium/idearium-lib/tree/master/docusaurus/',
```

Comment:
This is incorrect.

Reply:
@smebberson I put this url there for when the monorepo eventually gets merged into master, at that point it should be correct?

Reply:
Why don't we put in the `feature-monorepo` branch for now?